src/vde_backend/preprocess_sensors_config.py

```diff
@@ -335,10 +335,11 @@ def check_changes(c1_path, c1, c2_path, c2):
     nb_changes = 0
 
     if c1 and c2:
+        c1_hids = c1.get_home_sensors()
         for hid, sids_c2 in c2.get_home_sensors().items():
             print("{} : ".format(hid), end="")
             if hid in c1.get_home_sensors():
-
+                del c1_hids[hid]
```

Comment:
Not sure why you delete this variable before re-creating it with the same piece of code.
You could use `sids_c1 = c1_hids.pop(hid)`.

Reply:
The idea of this variable is to verify if we deleted a house or not. I will get houses that are deleted in the new config.